server/datastore/psql/maintenance_requests.go

```diff
@@ -0,0 +1,51 @@
+package psql
+
+import (
+	"time"
+
+	"github.com/a2fumn2022/gpmn-nsl/server/datastore"
+)
+
+// Import the necessary packages
+type rawTime []byte
+
+func (t rawTime) Time() (time.Time, error) {
+	return time.Parse("15:04:05", string(t))
+}
+
+func (db *DB) AllMaintenanceRequests() []datastore.MaintenanceRequest {
+
+	rows, err := db.connection.Query("select * from public.nsl_maintenance LIMIT $1", 777) //Number limits # of requests, can be anything
```

Comment:
I mention this in the Loom video, but I asked Isaac to update the table name to `maintenance_requests`. I guess I should've pushed this change too and mentioned in the video that we want to name our tables by the resource they're modeling. So a maintenance request would be stored in `maintenance_requests`, an exit checklist would be stored in `exit_checklists`, and so on.

Reply:
So please change this to `maintenance_requests`. (Also, you don't need the `public.` part of it)

Reply:
is there a reason you have this limit on the request?

also could you make the capitalization for queries consistent? The standard is to have all uppercase for sql keywords and lowercase for things like table names, fields, etc. So for example this query would be:

`SELECT * FROM public.nsl_maintenance LIMIT $1`

except changing it to the table names that joseph says to change to.

Reply:
ran up migrations and created a maintenence_request table, time.Time works now.